src/groups/mqb/mqbblp/mqbblp_queue.cpp

```diff
@@ -852,31 +852,8 @@ int Queue::processCommand(mqbcmd::QueueResult*        result,
     // executed by *ANY* thread
 
     if (command.isPurgeAppIdValue()) {
-        if (command.purgeAppId().empty()) {
-            mqbcmd::Error& error = result->makeError();
-            error.message()      = "Queue Purge requires a non-empty appId ("
-                                   "Specify '*' to purge the entire queue).";
-            return -1;  // RETURN
-        }
-
-        result->makePurgedQueues();
-        result->purgedQueues().queues().resize(1);
-        mqbcmd::PurgeQueueResult& purgedQueueResult =
-            result->purgedQueues().queues().back();
-
-        // Empty string means all appIds, however, for the command, we require
-        // the user to be explicit if the entire queue is to be deleted, and
-        // therefore require '*' for the appid.
-        bsl::string appId = (command.purgeAppId() == "*")
-                                ? ""
-                                : command.purgeAppId();
-        dispatcher()->execute(bdlf::BindUtil::bind(&Queue::purge,
-                                                   this,
-                                                   &purgedQueueResult,
-                                                   appId),
-                              this);
-        dispatcher()->synchronize(this);
-        return 0;  // RETURN
+        BSLS_ASSERT_SAFE(false && 
+                         "Should not get here. PURGE QUEUE command must be processed on a storage level.");
```

Comment:
Moved out purge queue command from `mqbblp::Queue`.
We should never get here.

And also there is a question, do we want to move 2 other commands from here, so `mqbblp::Queue` does not process commands at all.

TODO: remove purgeAppId from mqbcmd selection